### PR TITLE
Assembler generation

### PR DESCRIPTION
**Pull request by gv-ORC**
I have put all the supporting classes and everything inside `assembler_playground.js`.. its all in 1 file for now, but will pull it all apart.

On lines 115-148, I have defined an instruction set that has nop, add, and sub implemented. I think I need to do a few more things to work out the "addExecution" and similar functions, so the primary emulator knows what type of operands will be in the function... i think I need to add a class that allows you to define instruction format type... so that way sources and destinations and such are universal, and the opcode/operand classes I have now would be used as parts of those types... and this type would be used in the machineFormat and assemblyFormat sections.

### CODE REVIEW

**Comment by gv-ORC** [comment]
i am aware of some random syntax errors and spelling issues, fixed those since this commit.

**Comment by gv-ORC** [comment]
known ToDo: create classes for machineFormat and assemblyFormat that define the instruction type.

File: public/Instruction.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    assemblyFormat = [];
+    constructor (mnemonic, assembledLength, mf, af) {
+        super ();
+        this.assembledLength = assembledLength;
+        this.mnemonic = mnemonic;
+        this.machineFormat = mf;
+        this.assemblyFormat = af;
+    }
+    internalAssemble(program) {
+        return [this.mnemonic, program.next_line()];
```
Comment (Myvar) [inline public/Instruction.js:52]: Are you just doing this as a temp matter for now?

File: src/output.css
Comment (Data-Corruption) [inline src/output.css:1]: This file should be ignored.

**Comment by Data-Corruption** [comment]
You should probably keep unused / test code in a gitignored file or folder. That would make things a lot cleaner and easier to understand. Other than that and the output.css, without doing a deep dive and understanding any of the logic, this seems ok.

File: public/Instruction.js (hunk 1 above)
Comment (gv-ORC) [reply public/Instruction.js:52]: Yes, the assemblyFormat has yet to be setup as a class... havent touched the actual assemble functions yet :)

File: src/output.css
Comment (gv-ORC) [reply src/output.css:1]: I need to setup an ignore, will do it real quick later before I continue coding.

**Pull request merged**